specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/2017-11-15/timeseriesinsights.json

```diff
@@ -323,6 +323,43 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.TimeSeriesInsights/environments/{environmentName}/status": {
```

Comment:
Why isn't this returned as a property on the environment resource itself? Get on environments/{name} provides this data.

Reply:
The objective of the "environments/status" API is to provide the read-only status of various environment related operations. It does not directly describe the environment resource.  As of now, we expose the ingress status, in the future we will be adding the status of each of the 
 different environment related components (like Query) to the "/status" call. This will be a snapshot of statuses of each component that changes with time.

Reply:
@sergey-shandar, @ravbhatnagar and @simongdavies  - Can we get an update on this PR?